docs/source/providers/local.rst

```diff
@@ -0,0 +1,32 @@
+Adding a Local Provider
+=======================
+
+This section describes the local providers that are used for development and
+testing purposes. Local providers give Koku the ability to access test data
+without requiring a dependency on an external service, such as AWS S3 or a
+pre-existing OpenShift cluster (OKD).
+
+AWS Local Provider
+------------------
+These steps will allow you to configure a local provider for a Koku instance
+deployed into an OpenShift environment.
+
+#. Deploy an OKD cluster: ``make oc-up``
+#. Deploy Koku into the cluster: ``make oc-create-koku``
+#. Add a `persistent volume <https://docs.okd.io/latest/dev_guide/persistent_volumes.html>`_ to the Koku pod
+    Mount point: ``/tmp/koku``
```

Comment:
Do we have a template for this?  I admittedly have not yet used a local provider.